### DISCUSSION TITLE
Manually Denote a Patch

### DISCUSSION THREAD

**Discussion by robertlynch3** [chuot/rdio-scanner#114] [Ideas]
I have two systems, 1 is a TRS the other is UHF conventional. One of the TRS talkgroups is patched directly to a UHF frequency. 

Is there any note to manually denote that the patch, as trunk-recorder does not know its a patch.

**Comment by chuot** [comment]
You can add an additional `patches` field to your curl arguments like so, supposing you are using an upload script:

```
curl http://127.0.0.1:3000/api/trunk-recorder-call-upload \
     -F "key=${apikey}" \
     -F "audio=@${basename}.wav;type=audio/wav" \
     -F "meta=@${basename}.json;type=application/json" \
     -F "system=${1:-0}" \
     -F "patches=[123]"

```

`123` being the patched talkgroup.

**Reply by robertlynch3** [reply]
I am using the trunk-recorder upload plugin. Will that key work if they are two separate systems?

**Reply by chuot** [reply]
No, patched_talkgroup (patches) works only within the same system.

**Reply by robertlynch3** [reply]
Okay, thanks. I don't know if my use case is valid or not.
